examples/common/observation_utils.py

```diff
@@ -4,41 +4,43 @@
 
 import numpy as np
 
+def get_observation_components_modified_for_visualization(observation, observation_components_to_modify):
 
-def modify_observation_for_visualization(obs, observation_comoponents_to_modify):
-    
-    modified_obs = {}
-    for render_pass, camera_names in observation_comoponents_to_modify.items():
-        for camera_name in camera_names:
-            obs_render_pass = obs[camera_name]
-            assert len(obs_render_pass.shape) == 3  # width, height, #channels
-            assert obs_render_pass.shape[2] == 4    # 4 channels (RGBA)
+    modified_observation = {}
+    for modification_mode, observation_component_names in observation_components_to_modify.items():
+        for observation_component_name in observation_component_names:
 
-            if render_pass == "depth":
-                obs_render_pass_vis = obs_render_pass[:,:,[0,1,2]].copy() # depth is returned as RGBA
+            observation_component = observation[observation_component_name].copy()
+
+            if modification_mode in ["depth", "final_color", "normal", "segmentation"]:
+                assert len(observation_component.shape) == 3  # width, height, #channels
```

Comment:
Data not always returned as RGBA (sometimes it is returned as BGRA) so comment is inaccurate.
 
```
                assert len(observation_component.shape) == 3  # width, height, channels
                assert observation_component.shape[2] == 4    # 4 channels
```